managed/resource_read_replica.go

```diff
@@ -65,7 +68,19 @@ func (r resourceReadReplicasType) GetSchema(ctx context.Context) (tfsdk.Schema,
 					"vpc_id": {
 						Description: "The ID of the VPC where the read replica is deployed.",
 						Type:        types.StringType,
-						Required:    true,
+						Optional:    true,
+						Computed:    true,
+					},
+					"vpc_name": {
+						Description: "The name of the VPC where the read replica is deployed.",
+						Type:        types.StringType,
+						Optional:    true,
+						Computed:    true,
+						Validators: []tfsdk.AttributeValidator{
+							schemavalidator.ExactlyOneOf(path.MatchRelative().AtParent().AtName("vpc_name"),
```

Comment:
@shinji62 just for understanding and learning: why not `ConflictsWith` here also? Yeah, `ExactlyOneOf` can be used too, but is there any specific reason why you used it here?

Reply:
No you don't need to put in both, plus I use `ExactlyOneOf` because those (on of them) are required. `ConflictsWith` only check if you put those two field at the same time, but if any of them are present the validation pass.